collections/_version2-english/9-fork/2-to-dos.md

```diff
@@ -41,31 +51,17 @@ It cannot skip `universe B`.
- - The [official blog](https://augur.net/blog)
- - The subreddit [/r/Augur](https://www.reddit.com/r/Augur/)
-
 ## After starting a fork (60 days)
 REP holders needs to pick a side and migrate their REP to the [child universe]({{glossary_child_universe}}) they believe aligns with reality within 60 days from starting a [fork]({{glossary_fork}}).
 
 After 60 days from starting a fork, [REP]({{glossary_reputation_token}}) in the [parent universe]({{glossary_parent_universe}}) are [locked]({{glossary_locked_universe}}) and can not be migrated to the child universe permanently.
 
 However, the choice should be considered carefully, because migration is one-way; it cannot be reversed. REP cannot be sent from one [sibling universe]({{glossary_sibling_universe}}]) to another.
 
-### What if REP holders don't wish to join in the fork?
+## What if REP holders don't wish to join in the fork?
 They may sell their [REP]({{glossary_reputation_token}}) before the [fork]({{glossary_fork}}) is over, but beware that if many people exercise this strategy the price of REP may drop leading up to the fork and then rebound in the [truth universe]({{glossary_truth_universe}}) after the fork, this could be a costly strategy.
 
-### You receive 40% ROI by joining in the fork?
-Wether you receive 40% [ROI]({{glossary_roi}}) on your migrated [REP]({{glossary_reputation_token}}) by joining in the [fork]({{glossary_fork}}) depends on when and where you migrate your REP. If you migrate your REP during the [forking period]({{glossary_forking_period}}) and the [child universe]({{glossary_child_universe}}) which you migrate it becomes [*winning universe*]({{glossary_winning_universe}}) , then you get 40% ROI on your migrated REP. If you migrate after the forking period, you may not get 40% ROI but you can migrate it without risk of migrating to [*losing universe*]({{glossary_losing_universe}}). Because the [final outcome]({{glossary_final_outcome}}) of the [forking market]({{glossary_forking_market}}) is fixed when the forking period ends.
-
-In other words, you must migirate your REP within 60 days after a fork starts, but it is not necessarily to migrate it during the forking period. The forking period ends when the forking market enters the [finalized state]({{glossary_finalized_market}}), it can take less than 60 days. While, the period which you can migrate your REP is always fixed 60 days.
+## Is there any reward for migrating REP to a child universe?
+No. If you migrate your [REP]({{glossary_reputation_token}}) during the [forking period]({{glossary_forking_period}}), there is no reward for that. However, if you staked your REP on the [forking market]({{glossary_forking_market}})'s outcome before the fork starts and the outcome you staked on becomes the [final outcome]({{glossary_final_outcome}}) of the forking market, you will receive a 40% [ROI]({{glossary_roi}}) on your stake in the [winning universe]({{glossary_winning_universe}}).
```

Comment:
May want to mention again here that there is no reward, but there is a very severe punishment for *not* participating.

Reply:
I will add such a sentence as another PR, thanks.